tools/build_utils.py

```diff
@@ -291,7 +291,7 @@ def build_tensorflow_cc(tf_version,
         artifacts_dir,
         target_arch,
         verbosity,
-        target="//tensorflow:" + tf_cc_lib_name)
+        target="//tensorflow:" + tf_cc_lib_name + " tensorflow:install_headers")
```

Comment:
What for?

Reply:
It may not be necessary; let me double-check. The headers are no longer generated in `bazel-genfiles` in 2.2 and I wasn't sure if they're installed in `bazel-bin` by default or not.

Reply:
This is not necessary. Removed in 59b9e51.